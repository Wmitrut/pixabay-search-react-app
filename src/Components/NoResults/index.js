import Travolta from '../../Assets/travolta.gif';

const NoResults = (props) => {
  return (
    <div className="content-center justify-center text-center">
      <img
        src={Travolta}
        alt="John Travolta confused"
        className="object-cover m-auto my-2 w-96 h-64 rounded-lg"
      />
      <div className="content-center justify-center text-center">
        <span className="mt-6 text-gray-500">
          We cant find anything with <strong className="font-semibold">{props.searchTerm}</strong>,
          try searching again.
        </span>

        <button
          type="button"
          onClick={props.handleOnReset}
          disabled={props.searchTerm == ''}
          className="ml-2 rounded bg-indigo-400 py-2 px-4 font-bold text-white hover:bg-indigo-500 disabled:bg-indigo-500/20"
        >
          Clear Search
        </button>
      </div>
    </div>
  );
};

export default NoResults;
