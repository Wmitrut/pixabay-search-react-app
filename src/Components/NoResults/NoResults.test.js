import renderer from 'react-test-renderer';

import NoResults from './index';

it('Renders correctly without data', () => {
  const tree = renderer.create(<NoResults />).toJSON();
  expect(tree).toMatchSnapshot();
});
