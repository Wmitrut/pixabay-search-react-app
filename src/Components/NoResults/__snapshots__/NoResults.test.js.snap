// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly without data 1`] = `
<div
  className="content-center justify-center text-center"
>
  <img
    alt="John Travolta confused"
    className="m-auto my-2 h-64 w-96 rounded-lg object-cover"
    src="travolta.gif"
  />
  <div
    className="content-center justify-center text-center"
  >
    <span
      className="mt-6 text-gray-500"
    >
      We cant find anything with 
      <strong
        className="font-semibold"
      />
      , try searching again.
    </span>
    <button
      className="ml-2 rounded bg-indigo-400 py-2 px-4 font-bold text-white hover:bg-indigo-500 disabled:bg-indigo-500/20"
      disabled={false}
      type="button"
    >
      Clear Search
    </button>
  </div>
</div>
`;
